src/jit/compiler.cpp

```diff
@@ -4963,6 +4982,10 @@ void Compiler::compCompileFinish()
         // Make the updates.
         genMemStats.nraTotalSizeAlloc = compGetAllocator()->getTotalBytesAllocated();
         genMemStats.nraTotalSizeUsed  = compGetAllocator()->getTotalBytesUsed();
+#if MEASURE_MEM_ALLOC
+        memAllocHist.record((unsigned)((genMemStats.nraTotalSizeAlloc + 1023) / 1024));
+        memUsedHist .record((unsigned)((genMemStats.nraTotalSizeUsed  + 1023) / 1024));
```

Comment:
Please remove the space; this sort of alignment is not part of the coding convention.